README.md

```diff
@@ -49,7 +49,7 @@ Versioning conventions for Zowe CLI and Plug-ins| [Versioning Guidelines](./docs
 <br/>
 
 ## **Building Zowe CLI From Source**
-Zowe CLI requires NPM version 7 to install from source. Before proceeding, check your NPM version with `npm --version` and if it's older than 7.x, update with `npm install -g npm`.
+Zowe CLI requires NPM version 7 and Cargo version 1.72.0 (or newer) to build from source. Before proceeding, check your NPM version with `npm --version` and if it's older than 7.x, update with `npm install -g npm`. To check your version of Cargo, run `cargo --version`. Cargo can be installed using rustup: [https://rustup.rs/](https://rustup.rs/). To update Cargo, run the `rustup update` command.
```

Comment:
is it ok to modify the `npm` version number to at least 8? 
Nobody should be using anything lower than 9 at this point since Node started to bundle 9 with node 18 😋 
https://nodejs.org/en/download/releases

Reply:
Changed.